source/Wormhole.h

```diff
@@ -41,6 +44,9 @@ class Wormhole {
 	const std::string &Name() const;
 	// Whether this wormhole's link appears on the map.
 	bool IsMappable() const;
+	// Returns this wormhole's arrow color.
+	// The link color will be a dimmed version of this one.
+	const Color *GetLinkColor() const;
```

Comment:
If this is not actually the link colour, but the arrow colour, and the link colour is derived from this, should it really be called `GetLinkColor`?